ajax/libs/pushy/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pushy",
+  "filename": "js/pushy.min.js",
+  "description": "Pushy is a responsive off-canvas navigation menu using CSS transforms & transitions",
+  "version": "1.0.0",
+  "homepage": "https://www.christopheryee.ca/pushy/",
+  "author": "christopher <@cmyee>",
+  "keywords": [
+    "jquery",
+    "responsive",
+    "offcanvas",
+    "menu"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/christophery/pushy.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/christophery/pushy.git",
+    "basePath": "",
+    "files": [
+      "+(css|js)/pushy*",
+      "img/arrow.svg"
```

Comment:
If you'll like to use this schem, please also add `jpg` extension it that. Personally I prefer use a more flexible for img like `img/**/*`, images are usually not so messy as js/css files do.